diff -bur --exclude=CVS marc-record/lib/MARC/Field.pm marc-record/modified/MARC/Field.pm
--- marc-record/lib/MARC/Field.pm	Sun Mar 23 11:31:16 2003
+++ marc-record/modified/MARC/Field.pm	Sun Mar 23 11:15:00 2003
@@ -1,6 +1,5 @@
 package MARC::Field;
 
-use warnings;
 use strict;
 use integer;
 use Carp;
diff -bur --exclude=CVS marc-record/lib/MARC/File/MicroLIF.pm marc-record/modified/MARC/File/MicroLIF.pm
--- marc-record/lib/MARC/File/MicroLIF.pm	Sun Mar 23 11:55:38 2003
+++ marc-record/modified/MARC/File/MicroLIF.pm	Sun Mar 23 11:15:11 2003
@@ -8,7 +8,6 @@
 
 use strict;
 use integer;
-use bytes;
 
 ##MAKEFILE.PL_MACRO ($>=5.008) : use bytes;
 
diff -bur --exclude=CVS marc-record/lib/MARC/File/USMARC.pm marc-record/modified/MARC/File/USMARC.pm
--- marc-record/lib/MARC/File/USMARC.pm	Sun Mar 23 11:54:34 2003
+++ marc-record/modified/MARC/File/USMARC.pm	Sun Mar 23 11:15:21 2003
@@ -8,7 +8,6 @@
 
 use strict;
 use integer;
-use bytes;
 
 use vars qw( $ERROR );
 
diff -bur --exclude=CVS marc-record/lib/MARC/File.pm marc-record/modified/MARC/File.pm
--- marc-record/lib/MARC/File.pm	Sun Mar 23 11:31:16 2003
+++ marc-record/modified/MARC/File.pm	Sun Mar 23 11:15:30 2003
@@ -8,7 +8,6 @@
 
 use strict;
 use integer;
-use warnings;
 
 use vars qw( $ERROR );
 
diff -bur --exclude=CVS marc-record/lib/MARC/Lint.pm marc-record/modified/MARC/Lint.pm
--- marc-record/lib/MARC/Lint.pm	Sun Mar 23 11:31:16 2003
+++ marc-record/modified/MARC/Lint.pm	Sun Mar 23 11:15:37 2003
@@ -1,6 +1,5 @@
 package MARC::Lint;
 
-use warnings;
 use strict;
 use integer;
 
diff -bur --exclude=CVS marc-record/lib/MARC/Record.pm marc-record/modified/MARC/Record.pm
--- marc-record/lib/MARC/Record.pm	Sun Mar 23 11:31:16 2003
+++ marc-record/modified/MARC/Record.pm	Sun Mar 23 11:15:45 2003
@@ -6,7 +6,6 @@
 
 =cut
 
-use warnings;
 use strict;
 use integer;
 
