diff -bur --exclude=CVS marc-record/lib/MARC/Field.pm marc-record/new/MARC/Field.pm
--- marc-record/lib/MARC/Field.pm	Sun Mar 23 12:08:07 2003
+++ marc-record/new/MARC/Field.pm	Sun Mar 30 17:37:13 2003
@@ -1,6 +1,5 @@
 package MARC::Field;
 
-use warnings;
 use strict;
 use integer;
 use Carp;
diff -bur --exclude=CVS marc-record/lib/MARC/File.pm marc-record/new/MARC/File.pm
--- marc-record/lib/MARC/File.pm	Sun Mar 23 12:08:07 2003
+++ marc-record/new/MARC/File.pm	Sun Mar 30 17:37:24 2003
@@ -8,7 +8,6 @@
 
 use strict;
 use integer;
-use warnings;
 
 use vars qw( $ERROR );
 
diff -bur --exclude=CVS marc-record/lib/MARC/Lint.pm marc-record/new/MARC/Lint.pm
--- marc-record/lib/MARC/Lint.pm	Sun Mar 23 12:08:08 2003
+++ marc-record/new/MARC/Lint.pm	Sun Mar 30 17:37:35 2003
@@ -1,6 +1,5 @@
 package MARC::Lint;
 
-use warnings;
 use strict;
 use integer;
 
diff -bur --exclude=CVS marc-record/lib/MARC/Record.pm marc-record/new/MARC/Record.pm
--- marc-record/lib/MARC/Record.pm	Sun Mar 23 12:17:46 2003
+++ marc-record/new/MARC/Record.pm	Sun Mar 30 17:37:43 2003
@@ -6,7 +6,6 @@
 
 =cut
 
-use warnings;
 use strict;
 use integer;
 
